packages/utilities/psammead-test-helpers/src/index.js

```diff
@@ -5,7 +5,11 @@ import deepClone from 'ramda/src/clone';
 export const shouldMatchSnapshot = (title, component) => {
   it(title, () => {
     const { container } = render(component);
-    expect(container.firstChild).toMatchSnapshot();
+    const hasOneChild = container.children.length === 1;
+    const removeWrappingDiv = containerTwo => containerTwo.firstChild;
```

Comment:
Can you add  comment here explaining why please :)

Reply:
why `containerTwo`?

Reply:
@jroebu14 ESLint did not like that 'container' was already declared in the upper scope.

Reply:
ah ok. it should be fine if you move the `removeWrappingDiv` function to the line before `const { container } = render(component);`